### ISSUE TITLE
Meteor: Bitcode still set to true after installing this plugin

### ISSUE THREAD

**Issue by derwaldgeist** [akofman/cordova-plugin-disable-bitcode#6]
I tried this plugin in combination with the Meteor framework, but did not have success with it. During build, the log prints

```
✔ BITCODE disable
```

so I assume the script is running correctly. But if I open the generated project file in Xcode, the Bitcode setting is still set to `true`. Any ideas what might cause this?

**Comment by akofman** [comment]
Hello @derwaldgeist,

What version of this plugin are you using ?
Is it the only one you're using in your project ?
Could you have a try with the 1.2.1 version and let me know if it's better for you ?

Thanks ;)

**Comment by derwaldgeist** [comment]
Thanks for your quick reply!

I tried the 1.3.0 version, and I have several other plugins installed. I will try the 1.2.1 now and provide some feed-back on it.

**Comment by brauchle** [comment]
I just had the same problem.
It seems to work if you add the plugin after you added a platform to the project.
But if you added the plugin once and if you later remove the platform and re-add it again, the plugin does not change the value to NO.

**Comment by akofman** [comment]
Thanks for your comments guys. For those with this issue, I think you could use the 1.2.1 version of this plugin and that should be better. 

I'm going to take time to fixe that before the end of the week ;)

**Comment by derwaldgeist** [comment]
Thanks, that sounds promising - unfortunately, I had no time yet to try out the 1.2.1 version, but I will surely try out your fix once it's there.

**Comment by akofman** [comment]
Ok everyone, version 1.3.2 (the last one) should fix everything.

I checked multiple tests : 
- Installing a new platform then adding the plugin => OK
- Installing a new platform with the plugin configured in the config.xml => OK
- Removing the platform then adding it again but keeping the plugin already installed => OK

Please let me know if it helps !

**Comment by derwaldgeist** [comment]
Thanks, will check that out!

**Comment by brauchle** [comment]
Yep, now it works. Thanks for the fix.

**Comment by derwaldgeist** [comment]
@akofman I've checked it with Meteor now (sorry for the delay). It doesn't work, but that's not your fault. Your code is correct. Problem is that during the Cordova "prepare" step, all changes that have already been written to the Xcode project are suddenly reverted. I think this is a problem with Meteor's build scripts. I found this out when I tested my own hook and it didn't work either. Only after I changed the hook to "after_prepare" the changes were persistent. But this should not be necessary IMHO. Anyway, thanks for your plugin!
